rotkehlchen/chain/ethereum/modules/stakedao/utils.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from collections import defaultdict
+from typing import Final
+
+import requests
+
+from rotkehlchen.db.settings import CachedSettings
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.globaldb.cache import globaldb_set_general_cache_values
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import deserialize_evm_address
+from rotkehlchen.types import CacheType, ChainID
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+STAKEDAO_API: Final = 'https://api.stakedao.org/api'
+PROTOCOLS: Final = {'curve', 'balancer', 'angle'}
```

Comment:
https://github.com/stake-dao/api?tab=readme-ov-file#strategies-data
there are more strategies in the link above. why did you add just three?